chapter10/pxd/ex3.pl

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use feature "say";
+
+open my $fh, '<', 'words.txt' or die "Can't open file words.txt : $!";
+my @words = <$fh>;
+chomp(@words);
+
+my @sorted_list =
+    map $_->[0],
+    sort { $a->[1] cmp $b->[1] }
+    map [ $_->[0], $_->[1] =~ tr/a-z//cdr ],
+    map [ $_, tr/A-Z/a-z/r ], @words;
```

Comment:
You could do both these transforms in the same `map` so you have the 3 schwartzian stages (rather than 4).